modules/apps/27-interchain-accounts/types/keys.go

```diff
@@ -8,12 +8,15 @@ const (
 	// ModuleName defines the interchain accounts module name
 	ModuleName = "interchainaccounts"
 
-	// VersionPrefix defines the current version for interchain accounts
-	VersionPrefix = "ics27-1"
-
-	// PortID is the default port id that the interchain accounts module binds to
+	// PortID is the default port id that the interchain accounts host submodule binds to
 	PortID = "interchain-account"
 
+	// PortPrefix is the default port prefix that the interchain accounts controller submodule binds to
+	PortPrefix = "ics27"
```

Comment:
The controller port ID now is prefixed, rendering as: `ics27-cosmos1xjfs...` 
Happy to remove the `Delimiter` and define this as below if necessary:

```
PortPrefix = "ics27-"
```

Reply:
+1 to moving the `-`

Reply:
Done!